### ISSUE TITLE
Content Security Policy: The page's settings blocked the loading of a resource at inline ("script-src").

### ISSUE THREAD

**Issue by Southclaws** [vercel/next.js#48461]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

```bash
Platform: darwin
      Arch: arm64
      Version: Darwin Kernel Version 21.6.0: Wed Aug 10 14:28:35 PDT 2022; root:xnu-8020.141.5~2/RELEASE_ARM64_T8101
    Binaries:
      Node: 16.13.1
      npm: 8.1.2
      Yarn: 1.22.15
      pnpm: 6.11.0
    Relevant packages:
      next: 13.3.1-canary.8
      eslint-config-next: 13.3.0
      react: 18.2.0
      react-dom: 18.2.0
```

### Link to the code that reproduces this issue

https://github.com/Southclaws/storyden/pull/13

### To Reproduce

It seems that just setting up an app with the new app directory causes this. There's no clear resolution in any of the error messages.

### Describe the Bug

"Content Security Policy: The page's settings blocked the loading of a resource at inline ("script-src")."

And no content will load

<img width="1192" alt="image" src="https://user-images.githubusercontent.com/1636971/232340004-40854db6-5e39-45fa-ad73-e3da38703da1.png">

### Expected Behavior

I was expecting the upgrade to not do anything that affected or required changes to my site's CSP.

### Which browser are you using? (if relevant)

Firefox latest

### How are you deploying your application? (if relevant)

Vercel

**Comment by juinquok** [comment]
Seems to be the same bug as https://github.com/vercel/next.js/issues/43743

Getting this error on my App as well

**Comment by juinquok** [comment]
Seems to be the same bug as https://github.com/vercel/next.js/issues/43743

Getting this error on my App as well

**Comment by Southclaws** [comment]
This appears to be solved on the latest (13.4.4)
